### PR TITLE
CVE-2011-1183 research

### PR DESCRIPTION
**Pull request by dajanio**
Initial commit of CVE-2011-1183 research updates

### CODE REVIEW

File: cves/CVE-2011-1183.yml [hunk 1]
```diff
@@ -44,12 +44,8 @@ description_instructions: |
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
 description: |
-  The basic configurations for a Tomcat web application are set in the web.xml file. 
-  This file includes a config called "login-config" which sets a level of security 
-  for the application by whitelisting url's based on a user's role. This bug occurred 
-  when this configuration was not set and a configuration called "metadata-complete" 
-  was set to true, unintentionally allowing users to "bypass intended access restrictions" 
-  through HTTP requests.
+  A configuration file, web.xml usually contains a login configuration. When this file has no login configuration,
```
Comment (dwelcheck) [inline cves/CVE-2011-1183.yml]: Put a comma after `web.xml` to help delineate where that clause ends.

File: cves/CVE-2011-1183.yml [hunk 2]
```diff
@@ -65,15 +61,17 @@ fixes_vcc_instructions: |
   was discovered in the "note" field.
   Refer to our instructions on how to find a Git SHA from an SVN revision.
 fixes:
-   - commit: b7b5c63a932f6c1ea05f9b65ad9054247bb5af57
-     note: SVN rev 1087643, from the Tomcat website.
-   - commit:
-     note:
+  - commit: b7b5c63a932f6c1ea05f9b65ad9054247bb5af57
+    note: Extra logic added to check for the empty login config.
+  - commit:
```
Comment (dwelcheck) [inline cves/CVE-2011-1183.yml]: If there was not an additional fix commit, remove this block entirely.

File: cves/CVE-2011-1183.yml [hunk 3]
```diff
@@ -106,8 +104,8 @@ unit_tested:
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
   answer: |
-    The function where the fix exists was not unit tested until the new fix was 
-    implemented. A test was then added after the new fix.
+    While tests did exist for this particular portion of the code, none were specifically looking for a problem
+    like this. After the CVE was fixed a new test was written specifically for this scenario.
   code: false
```
Comment (dwelcheck) [inline cves/CVE-2011-1183.yml]: I believe that since the original code _was_ unit tested, and this edge case was not accounted for, the answer would be `true`.

**Review by dwelcheck — COMMENTED** [review]
+3 upvotes
